frontend/src/index.pug

```diff
@@ -184,18 +205,12 @@ html
               .summary-charts__title--repo {{ repo[0].displayName }} ({{ repo[0].name }})
               .summary-charts__title--contribution.mini-font(title="Total contribution of group") [{{ getGroupTotalContribution(repo) }} lines]
             .summary-charts__fileType--breakdown(v-if="filterBreakdown")
-              template(v-if="filterGroupSelection === 'groupByRepos'")
-                .summary-charts__fileType--breakdown__legend(
-                    v-for="fileType in getFileTypes(repo)")
-                  i.fas.fa-circle(
-                      v-bind:style="{ 'color' : contributionBarFileTypeColors[fileType] }"
-                  )
-                  span &nbsp; {{ fileType }} &nbsp;
-              template(v-else)
+              template(v-if="filterGroupSelection !== 'groupByNone'")
                 .summary-charts__fileType--breakdown__legend(
-                    v-for="fileType in Object.keys(contributionBarFileTypeColors)")
+                  v-for="fileType in getFileTypes(repo)" :key="fileType"
```

Comment:
is it possible to standardise the `v-bind` usage? Rather than using the shorthand